### PR TITLE
adds hosted apps list

### PR DESCRIPTION
**Pull request by kilbergr**
# [Compile a list of all the applications within our hosting purview.](https://github.com/OHS-Hosting-Infrastructure/infrastructure/issues/34)

Changes proposed in this pull request:

- Add a chart of currently running EC2 instances and associated team names.

### CODE REVIEW

**Comment by jduss4** [comment]
This is so helpful!!!  Thanks Rebecca

File: docs/how-we-work/hosted-apps.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| SFTP                    | i-090980b098659acfc |             |                    |           |
+| Ansible-Nagios          | i-0a4fdf046f3f65f40 |             |                    |           |
+| ZTT-Notifications-Dev   | i-080050c40495fd67e | Dev         | Zero to Three      |           |
+| ZTT-Notifications       | i-0ceab680e61101e4e | Prod        | Zero to Three      |           |
+| ZTT-Apps-Dev            | i-02364bce86ac01473 | Dev         | Zero to Three      |           |
+| ZTT-Apps                | i-0cebb7cbc4661bc24 | Prod        | Zero to Three      |           |
+| ZTT-NAT-Dev             | i-0a4b60f937c50a3d3 | Dev         | Zero to Three      |           |
+| ZTT-NAT                 | i-07ff7ba294fed4a88 | Prod        | Zero to Three      |           |
+| Qualys                  | i-0a3d00ba2b2f9fc54 |             | HSICC              |           |
+| InjuryIllness           | i-0ffd5609ef36a8230 | Prod        | NCHBHS             | On hiatus |
```
Comment (rahearn) [inline docs/how-we-work/hosted-apps.md:38]: What does "On hiatus" mean? Is it still running?

**Review by rahearn — COMMENTED** [review]
Nice! I'm curious/terrified that there are so many machines without a team set.

Would it be useful to note the instance size and/or billing status? I know at least some of these machines are on full-up-front reserved instances, but it'd be nice to know if there's any cost savings to be had from shutting down any that aren't being used, or when the reserved contract ends.

File: docs/how-we-work/hosted-apps.md (hunk 1 above)
Comment (kilbergr) [reply docs/how-we-work/hosted-apps.md:38]: I've asked Alana in slack--I honestly don't know exactly. https://trussworks.slack.com/archives/C01T6F5PB1A/p1623180327079800
We'll see what she says.

Comment (kilbergr) [reply docs/how-we-work/hosted-apps.md]: "It means it never was up and running. It was all loaded and ready to go, but had no documentation, and then the vendor teams changed."

**Comment by rahearn** [comment]
> I think Ryan means whether these are reserved instances (whether they're prepaid for by the year/2year/etc) or if we're spending based off of hourly pricing.

Yes, that's what I meant. Even better than adding that to this doc would be a link to somewhere that was always up to date, if that exists in the console.

**Comment by kilbergr** [comment]
OK so I went to this page: https://console.aws.amazon.com/billing/home?#/bills?year=2021&month=6 and I believe what it's telling me is that none of our EC2 instances are RI (reserved instances) so that means they are ALL hourly billing.
Screen Shot 2021-06-10 at 3 59 41 PM
Screen Shot 2021-06-10 at 4 00 05 PM
I'll link to this in here but I believe that's your answer--I'll include generally rather than using an extra column since they're all the same.

**Comment by rahearn** [comment]
Some of the EC2 instances are on demand, as your screenshot shows, but there are also a bunch of reserved instances in use shown just below where your screenshot cuts off:

Screen Shot 2021-06-14 at 4 30 02 PM

But that view doesn't give a great look at when each of those RI contracts expires. AWS Cost Management _might_ help, but I haven't used it enough to know for certain.

**Comment by kilbergr** [comment]
I saw that but since there was $0 in the rightmost column, I thought that meant they weren't used this cycle. I can't actually tell what the various columns look like. 
I'll look into AWS Cost Management to see if I can find something more.

**Comment by rahearn** [comment]
> since there was $0 in the rightmost column, I thought that meant they weren't used this cycle

that's just because they are fully upfront plans instead of partial up front plans.

**Comment by kilbergr** [comment]
@rahearn ok, from what I understand: reserved instances aren't dedicated to a single machine all the time. They get assigned as they are available. That's why you can find the reservation ID via the console. It will be tied to a specific instance when in use by that instance, but the instance could change. I guess in a system where machines don't spin up and down frequently, you could get a pretty clear idea of whether the instances were using reserved machines, but they could still change. 

I've been able to find reserved instances via `aws ec2 describe-reserved-instances --region us-east-1` command and to find a reservation id by looking at `aws ec2 describe-instances --region us-east-1`, but the reserved instance id != a reservation id.
At this point, I think it is not worth pursuing here, as I believe it's out of the scope of this ticket. Would you mind writing a ticket for our backlog instead describing what this information would be used for?

**Comment by rahearn** [comment]
> At this point, I think it is not worth pursuing here, as I believe it's out of the scope of this ticket. Would you mind writing a ticket for our backlog instead describing what this information would be used for?

I agree, and I will write a new ticket.

Good point about the reservations not being tied directly to the instances. I think in this use they are pretty stable, but manually updating docs when things do change is not the best use of anyone's time.

**Comment by kilbergr** [comment]
Awesome. Thanks, @rahearn ! Will you approve this 🙏 ?

**Review by rahearn — APPROVED** [review]

**Pull request merged**